kustomize/base/nginx/templates/routes/openai.conf.template

```diff
@@ -6,6 +6,7 @@ location /openai {
       rewrite ^\/openai(\/.*)$ $1 break;
       proxy_pass ${NGINX_UPSTREAM_OPENAI};
       proxy_set_header Connection '';
+      proxy_set_header Authorization $openai_http_authorization;
```

Comment:
this is to make it work with any openai that requires authentication, with this, when the header is `AGENT_MORPHEUS` it will be replaced with the configured env var, if the authentication header is present in the request and has a different value, this value will be used instead.
It is configured in the same way for other services that require authentication and for some reason was missing here.